modules/integration_aws-backup/README.md

```diff
@@ -57,7 +58,7 @@ Note the following parameters:
 
 These 3 parameters alongs with all variables defined in [common-variables.tf](common-variables.tf) are common to all
```

Comment:
```suggestion
These 4 parameters along with all variables defined in [common-variables.tf](common-variables.tf) are common to all
```

Reply:
Never mind, I did not see  the folded section.

Also, the `alongs` typo is everywhere...

Reply:
> Note the following parameters:
> * `source`: [...]
> * `environment`: [...]
> * `notifications`: [...]
> These 3 parameters [...] are common to all modules in this repository. Other variables, specific to this module, are available in variables.tf [...].

Looks good to me, since the "3" does not refer to the code example but the list just above. The source template needs to be fixed however, doing another PR for the typo since it will affect every stacks.

Reply:
Yes, that's why I said _Never mind_ :)

Reply:
https://github.com/claranet/terraform-signalfx-detectors/pull/510

Reply:
Github UI lagging, I wasn't seeing your comment, until I reload the page :sweat_smile: